mungegithub/mungers/approval-handler.go

```diff
@@ -65,15 +71,16 @@ func (*ApprovalHandler) AddFlags(cmd *cobra.Command, config *github.Config) {}
 
 // Munge is the workhorse the will actually make updates to the PR
 // The algorithm goes as:
-// - Initially, we set up approverSet
-//   - Go through all comments after latest commit. If any approver said "/approve", add him to approverSet.
-// - For each file, we see if any approver of this file is in approverSet.
+// - Initially, we build an approverSet
+//   - Go through all comments after latest commit.
+//	- If anyone said "/approve", add them to approverSet.
+// - Then, for each file, we see if any approver of this file is in approverSet and keep track of files without approval
 //   - An approver of a file is defined as:
-//     - It's known that each dir has a list of approvers. (This might not hold true. For usability, current situation is enough.)
-//     - Approver of a dir is also the approver of child dirs.
-//   - We look at top N (default 3) level dir approvers. For example, for file "/a/b/c/d/e", we might search for approver from
-//     "/", "/a/", "/a/b/"
-// - Iff all files has been approved, the bot will add "approved" label.
+//     - Someone listed as an "approver" in an OWNERS file in the files directory OR
+//     - in one of the file's parent directorie
+// - Iff all files have been approved, the bot will add the "approved" label.
+// - Iff the approved label has been added and a cancel command is found, that reviewer will be removed from the approverSet
```

Comment:
Why does the label need to be set in order for them to be removed from the approverSet?

Can I 'disapprove' something someone else approved?